atomist/integrate/github.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Connect to GitHub
+description:
+keywords:
+---
+
+{% include atomist/disclaimer.md %}
+
+When installed for a GitHub organization, the Atomist GitHub app links
+repository activity to container images.
+
+Install the GitHub app in the organization that contains the source code
+repositories for your Docker images.
+
+## Connect to GitHub
+
+1. Go to <https://dso.docker.com/> and sign in using your Docker ID.
```

Comment:
If someone has already added atomist to a github organisation but not in this team then the flow is slightly different and we will allow them to select it directly on this page (indeed we actually  can't send them via github due to the way the github app works). I don't know whether or not this is worth documenting here but it has been a persistent source of confusion and UI pain when it comes up. Mostly it seems to be an issue for internal users who are likely to have a number of teams.

Reply:
I would suggest that we leave this for now and revisit this later if it does come up as a nuisance in customer feedback